components/announcement_bar/version_bar/version_bar.tsx

```diff
@@ -5,32 +5,31 @@ import React from 'react';
 
 import {FormattedMessage} from 'react-intl';
 
-import {equalServerVersions} from 'utils/server_version';
 import {AnnouncementBarTypes} from 'utils/constants';
 
 import AnnouncementBar from '../default_announcement_bar';
 
 interface Props {
-    serverVersion: string;
+    buildHash: string | undefined;
```

Comment:
Minor nit, but this could be shorthanded as
```suggestion
    buildHash?: string;
```

Reply:
Thanks a lot! Addressed that in both places